### ISSUE TITLE
The parser doesn't support responses without body

### ISSUE THREAD

**Issue by izzyblues** [hmoragrega/openapi-parser#1]
Here a snippet of the spec:
```
openapi: 3.0.3
info:
  title: Title
  description: Title
  version: 1.0.0
servers:
  - url: 'https'
paths:
  /notify:
    post:
      summary: Notify
      description: |
        Notify a new payment
      tags:
        - Payments
      operationId: Notify Payment
      requestBody:
        required: true
        content:
          application/json:
            schema:
              $ref: '../openapi.yml#/components/schemas/Notification'
      responses:
        '202':
          description: |
            Notification accepted
```
This is the error that I get:
```
cannot prepare endpoints data: cannot group endpoints by tag: no content type defined for response: Notification accepted
```

**Comment by hmoragrega** [comment]
Thank you for the report kind sir, the problem should have been fixed in release v0.9.2

**Comment by izzyblues** [comment]
I confirm that the issue is solved
